### PR TITLE
Explicitly exclude superfluous activity types when a user shares logs

### PR DESCRIPTION
**Pull request by stevebaranski**
## Description
Fixes the first portion of #9780, which simplifies how a user shares logs via the Support interface. I also took the liberty to alphabetize the existing contents of this folder.

## To test:
- Verify that the existing branch builds & tests pass.
- Run the branch and confirm that you see something similar to the screenshots below. One callout, depending on what's enabled in `...`, you may see more options.

_Before_
9870-1-before

_After_:
9870-1-after

### CODE REVIEW

**Comment by ScoutHarris** [comment]
Hey @stevebaranski !

This works as advertised. Just a couple questions:

1) I don't suppose it's possible to disable the 'WordPress' and WP 'Save as Draft' share extensions?
2) I'm wondering if we really need to remove the email ability. To me, it seems like a legit thing a user might want to do...

**Comment by stevebaranski** [comment]
@ScoutHarris thanks again for considering this PR! I'll respond to each of your questions in turn:

> I don't suppose it's possible to disable the 'WordPress' and WP 'Save as Draft' share extensions?

`UIActivityViewController` allows us to specify `excludedActivityTypes`. I don't think the extensions are exposed in the same way. That said, we could exclude [`SharePost`](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Shared/SharePost.swift#L14). I can amend the PR to specify that if you'd like.

> I'm wondering if we really need to remove the email ability. To me, it seems like a legit thing a user might want to do...

The "legacy" instructions (from Helpshift) instruct the user to tap _Copy_ so the log data can be pasted into the ticket. I thought email might be more viable if we had a specific Zendesk mail address that could route a message (with logs) to a ticket. A power-user could still copy the log data, then fast-app switch to `Mail.app`. 

Thoughts?

**Comment by ScoutHarris** [comment]
> we could exclude SharePost

Makes sense to me. It seems odd someone would want to post the logs.

> I thought email might be more viable if we had a specific Zendesk mail address that could route a message (with logs) to a ticket.

I understand that's another suggested change on the original issue. I was really just wondering if it was necessary to remove the email option at this point. I was just thinking it sounded reasonable to me a user might want to email logs in general for various reasons. Removing a seemingly valid option just made me wonder. I'm not stuck on it though. Go with your gut! 😄

**Comment by stevebaranski** [comment]
@ScoutHarris I have pushed an additional commit that **excludes** the WP Share activity type and **includes** Mail. If & when we proceed with "phase 2" of this change, we might exclude the mail activity at that time.

When you have a moment, would you mind taking a look?

File: WordPress/Classes/ViewRelated/Me/ActivityLogDetailViewController.m [hunk 1]
```diff
@@ -58,18 +60,70 @@ - (void)viewWillAppear:(BOOL)animated
...
+
+    [activityTypes addObject:[SharePost activityType]];
+
+    return activityTypes;
+}
+
+/**
+ Specifies the activity types that should be excluded when the view controller is presented.
+
+ @return in practice, this should only return `UIActivityTypeCopyToPasteboard` & `UIActivityTypeMail`.
```
[the remark this answers is not in the document]
Comment (stevebaranski) [reply WordPress/Classes/ViewRelated/Me/ActivityLogDetailViewController.m]: Absolutely! There's no sense in adding documentation if it's misleading or incorrect. I'll make this change at once.

**Pull request merged**